@autoantwort ,This is the error log:
````
-- Building x64-linux-rel
-- Static build: Removing the full bin directory.
CMake Error at scripts/cmake/vcpkg_replace_string.cmake:12 (file):
  file failed to open for reading (No such file or directory):

    /mnt/vcpkg-ci/packages/plplot_x64-linux/share/plplot/examples/plplot-test.sh
````

--- a/ports/plplot/portfile.cmake
+++ b/ports/plplot/portfile.cmake
@@ -47,10 +47,6 @@ vcpkg_copy_pdbs()
 
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/plplot)
 
-if(EXISTS "${CURRENT_PACKAGES_DIR}/share/plplot/examples/plplot-test-interactive.sh")
-    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/plplot/examples/plplot-test-interactive.sh" "${CURRENT_PACKAGES_DIR}/share/plplot" "../")
-endif()
-
 # Remove unnecessary tool
 file(REMOVE
     "${CURRENT_PACKAGES_DIR}/debug/bin/pltek.exe"
@@ -70,10 +66,7 @@ endif()
 # Remove unwanted and duplicate directories
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
-
-if(NOT VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/plplot/examples/plplot-test.sh" "${CURRENT_BUILDTREES_DIR}" "")
-endif()
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/plplot/examples")
 
 file(INSTALL "${SOURCE_PATH}/Copyright" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
 

--- a/versions/p-/plplot.json
+++ b/versions/p-/plplot.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "93267fb900d6f13cee9b3200551037836deb1834",
+      "git-tree": "9e9bda9b45cefac69d08bbb2f3e947deab8cac14",
       "version-semver": "5.13.0",
       "port-version": 14
     },

